src/main/java/uk/gov/hmcts/darts/authorisation/service/impl/AuthorisationServiceImpl.java

```diff
@@ -86,8 +86,8 @@ public Optional<UserState> getAuthorisation(String emailAddress) {
                                  securityRole.get(SecurityRoleEntity_.id),
                                  securityRole.get(SecurityRoleEntity_.roleName),
                                  securityPermission.get(SecurityPermissionEntity_.id),
-                                 securityPermission.get(SecurityPermissionEntity_.permissionName)
-                             )
+                                 securityPermission.get(SecurityPermissionEntity_.permissionName),
+                                 root.get(UserAccountEntity_.active))
```

Comment:
can we group this together with the other selections for `UserAccountEntity`? (lines 82 and 83). Just to keep things organised